contracts/DefaultStrategy.sol

```diff
@@ -364,23 +310,16 @@ contract DefaultStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Ma
             revert GroupAlreadyAdded(group);
         }
 
-        if (deprecatedGroups.contains(group)) {
-            if (!deprecatedGroups.remove(group)) {
-                revert FailedToRemoveDeprecatedGroup(group);
-            }
-        }
-
         if (
-            activeGroups.length() + deprecatedGroups.length() >=
+            (specificGroupStrategy.getSpecificGroupStrategiesLength() +
```

Comment:
Unfortunately if we will use election.getGroupsVotedForByAccount() this will list all the groups and we could run out of gas when we will enable voting for more then 10 validators.

I considered this as non issue since we want to enable voting over max number of groups anyways.

@m-chrzan What do you think ?